src/components/App.js

```diff
@@ -1,77 +0,0 @@
-import React, { Component } from 'react';
-import {connect} from 'react-redux'
-import {bindActionCreators} from 'redux'
-
-import {checkLocation, echoWebSocket, testLatency, getWeather} from '../actions/main'
-
-import WeatherCanvas from './WeatherCanvas'
-
-import css from '../style/main.less'
-
-class App extends Component {
```

Comment:
React handles it's own virtual DOM of elements so in that manner , it can work dom alterations faster and then push them to the real DOM . extending to Component means that the class can be embedded into reacts ecosystem. 
Extending Component also brings additional inherited methods to handle component flow on the virtual dom.